### PR TITLE
grading scene optimizer project canceled

### PR DESCRIPTION
**Pull request by pierreglibert**
remove files src

### CODE REVIEW

**Comment by deltakosh** [comment]
Oh what happened?

**Comment by pierreglibert** [comment]
Hi @deltakosh :)
I posted the raison here : http://www.html5gamedevs.com/topic/31108-assetmanager-sceneoptimizer-contribution/?page=3&tab=comments#comment-205794

**Comment by deltakosh** [comment]
Sorry about that

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,8 @@
 node_modules
 .tempChromeProfileForDebug
 *.map
-*.js
+/gradingSceneOptimizer/js/babylon.js
```
Comment (deltakosh) [inline .gitignore]: Can you revert changes in this file?

**Comment by pierreglibert** [comment]
revert done ;)

**Pull request merged**